packages/http-cache-storage-lru/lib/HttpCacheStorageLru.ts

```diff
@@ -0,0 +1,34 @@
+import type { IHttpCacheStorage, IHttpCacheStorageValue } from '@comunica/types';
+import * as LRU from 'lru-cache';
+
+export class HttpCacheStorageLru implements IHttpCacheStorage {
+  private readonly cache: LRU<string, IHttpCacheStorageValue>;
+
+  public constructor(args: IHttpCacheLruArgs) {
+    this.cache = new LRU({ max: args.max });
```

Comment:
Since this LRU cache can also independently evict cache entries once the cache becomes full, we probably also want to link the `dispose` method of this cache to the HTTP invalidation bus.